template_content/{% if use_python_pytest %}tests{% endif %}/test_helloworld.py.jinja

```diff
@@ -0,0 +1,34 @@
+import pytest
```

Comment:
I think the filename here should also use `{{ contract_name }}`?

Reply:
done